Potestas/Potestas/Observations/FlashObservation.cs

```diff
@@ -19,16 +20,93 @@ namespace Potestas.Observations
     * Why immutable structure is used here?
     * TESTS: Cover this structure with unit tests
     */
-    public struct FlashObservation : IEnergyObservation
+    public struct FlashObservation : IEnergyObservation, IEquatable<FlashObservation>
     {
-        public Coordinates ObservationPoint { get; }
+        public static readonly double precision = 0.01;
```

Comment:
I believe they usually use CAPITAL case or at least PascalCase for the public constants

Reply:
fixed